templates/invoices/new.html

```diff
@@ -375,6 +376,11 @@ <h2>Fakturadelar</h2>
                     this.expense_parts.splice(i, 1)
             }
         }
-    })
+    });
+
+    function updateUploaded() {
+    	const files = [...document.getElementById('file').files];
```

Comment:
```suggestion
        const files = [...document.getElementById('file').files];
```
Use spaces instead of tabs, for consistency

Reply:
Fixed